### ISSUE TITLE
deelmob: locale=es causes js exception

### ISSUE THREAD

**Issue by onaseef** [onaseef/yomobi#341]
if you log into www.deelmob.com aatest24@mailinator.com 123123  in english, it's fine.  if you then click on Espanol at the bottom, there's a js exception in builder-app.js line 452. It says: Cannot read property 'CompanyLogo' of undefined.

**Comment by gilbert** [comment]
I'm seeing this too. I think it has something to do with the recent es.yml changes. I'm guessing something got indented wrong somewhere, and it's throwing the whole file off.

**Comment by gilbert** [comment]
...which doesn't make that much sense because it works locally. Hm.

**Comment by onaseef** [comment]
It works locally for you too?

**Comment by gilbert** [comment]
Yes. Very strange.

**Comment by onaseef** [comment]
Is there away to push a change to staging without first having to save it to git? I was going to experiment with the es.yml file to see where the problem might be but i don't want to keep pushing variations of the file into git. It clutters up the history.

**Comment by gilbert** [comment]
I think the best way to do this is to create a new branch off premium, then copy/paste the final changes from the temporary branch into the premium branch.

**Comment by gilbert** [comment]
I'm sure you know this, but just in case, you can push the new branch to staging using:

```
$ git push staging new_branch_name:master
```

**Comment by onaseef** [comment]
Okay, so I swapped en.yml and es.yml and added fr.yml, and it's not the language file that's the issue. For some reason, when the locale switches between en and es, it seems the bhelp variable fails. If I switch to locale=fr, nothing happens at all.  You can see this behavior on 

http://www.deelmob.com   aatest24@mailinator.com   123123

**Comment by onaseef** [comment]
Okay, so I added fr to the list of accepted languages, and on my local server it recognizes french and accepts it. On the test server, if I set the locale=fr, it reverts to english. If I set it to locale=es it breaks, and if I set it to locale=en, it is fine.

**Comment by gilbert** [comment]
I've finally discovered the problem. It lies in this commit:

https://github.com/onaseef/yomobi/commit/b797a0e92508fcc8dc8405ad04c38c215caec384

I rolled back to the commit before this one, and `es` worked. Moving up to this commit itself, it stopped working.

See the first change in the link. Notice how there's no difference between the red and the green? I found this very suspicious. Looking at `git show`, I saw a hidden utf-16 character:

http://ge.tt/6RV659J/v/0

Upon some research, this character is called a BOM. I'm still looking into how to resolve this.
